hack/generate-seed-crds.sh

```diff
@@ -44,6 +44,9 @@ get_group_package () {
   "resources.gardener.cloud")
     echo "github.com/gardener/gardener/pkg/apis/resources/v1alpha1"
     ;;
+  "operator.gardener.cloud")
```

Comment:
Should we rename this file to `hack/generate-crds.sh` since the `operator.gardener.cloud` group and probably more groups like `druid.gardener.cloud` and `resources.gardener.cloud` won't only be related to seed clusters.